packages/proxy/lib/http/index.ts

```diff
@@ -289,7 +305,8 @@ export class Http {
       },
     }
 
-    const onError = () => {
+    const onError = (error: Error): Promise<void> => {
+      ctx.error = error
```

Comment:
Are there places where we are currently setting `ctx.error` where we no longer need to set `ctx.error` due to this change?

Reply:
whoops nvm, I realize now that this is how it worked before

Reply:
I think it was just above on line 181?